Chess-On-Chain.md

```diff
@@ -0,0 +1,255 @@
+
+FTE: 1,5
+
+Costs: 20,000 USD
+
+exclamation The default deliverables 0a-0d below are mandatory for all milestones, and deliverable 0e at least for the last one.
+
+| Number |Deliverable  | Specification |
+| -------- | -------- | -------- |
+|0a     | License   | Apache 2.0 / GPLv3 / MIT / Unlicense   |
+| 0b   | Documentation | We will provide a repository with  documentation of the pallets and a basic "How TO Guide " that explains how a user can (for example) spin up one of our Substrate nodes and earn tokens from gameplay , Step by Step guide to spawn up wallets , play chess and Earn tokens          |
+| 0c  |Testing and Testing out  |The code will have unit-test coverage  to ensure functionality . In the guide, we will describe how to run these tests   |
+| 0d   | Docker  :      | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone.    |
+|  0e  | Article :  |  We will write an article on each pallet and build in this milestones    |
+|  1   | OFC Pallet:  | implementation of  offchain workers Pallet to get data from Game played on ChessOnchain As JSON files with http requests  Offchain worker pallet  will collect Game data such as moves , winning streaks ,ranking and rating |
+|  2.   | Token Pallet:   |Substrate module will have a token supply pallet that will have an earn algorithm from checkmate NFTS and winning streaks    |
+|  3.   | Game UI:   |We will deliver chess game UI with javascript language  UI for  several features( 1v 1, bughouse , Handicap etc )  |
+
+
+
+
+
+
+
+
+Milestone 2 Example — Additional features
+
+Estimated Duration: 1 month
+
+Costs: 10,000 USD
+
+| Number |Deliverable  | Specification |
+| -------- | -------- | -------- |
+|0a     | License   | Apache 2.0 / GPLv3 / MIT / Unlicense   |
+| 0b   | Documentation | We will provide a repository with  documentation of the pallets and a basic "How TO Guide " that explains how a user can (for example) spin up one of our Substrate nodes and earn tokens from gameplay , Step by Step guide to spawn up wallets , play chess and Earn tokens          |
+| 0c  |Testing and Testing out  |The code will have unit-test coverage  to ensure functionality . In the guide, we will describe how to run these tests   |
+| 0d   | Docker  :      | We will provide a Dockerfile(s) that can be used to test all the functionality delivered with this milestone.    |
+|  0e  | Article :  |  We will write an article on each pallet and build in this milestones    |
+|  1   | Storage Pallet |Implement Storage Pallets  for game Data from offchain workers pallet  to substrate chain  with storage maps. to store for each user/player   |
+|  2.   | Chess Onchain Democracy & Governance Pallet :    |Implememnt and Work on Democracy Pallet for Governance    |
+|  3.   | 	Leader Board Pallet :     |A Decentralized Leaderboard Pallet for competetions and Records  of championship games , and different chess game results     |
```

Comment:
Please add architecture and/or flow diagrams to show what data and functionality each provides and how the data flows.